.pre-commit-config.yaml

```diff
@@ -0,0 +1,36 @@
+repos:
```

Comment:
pre-commit now becomes the source of linting.  note that pre-commit runs in a separate environment, so you're both telling it how to construct that environment here, as well as giving it args for various commands.  I like to put as few args in here as possible though, and leave all the configuration for pyproject.toml